var/spack/repos/builtin/packages/openturns/package.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Openturns(CMakePackage):
+    """OpenTURNS is a scientific C++ and Python library featuring an
+    internal data model and algorithms dedicated to the treatment of
+    uncertainties. The main goal of this library is to provide all
+    functionalities needed to treat uncertainties in studies with
+    industrial applications. Targeted users are all engineers who want
+    to introduce the probabilistic dimension in their so far
+    deterministic studies."""
+
+    homepage = "https://openturns.github.io/www/"
+    git      = "https://github.com/openturns/openturns.git"
+    maintainers = ['liuyangzhuan']
+
+    version('master', branch='master')
+
+    variant('python',   default=True,  description='Build Python bindings')
+
+    extends('python', when='+python')
+
+    depends_on('mpi', type=('build', 'run'))
+    depends_on('lapack', type=('build', 'run'))
```

Comment:
These kinds of dependencies are usually link dependencies. Does the package not link to any of these libraries? I don't think lapack has any executables to add to the PATH does it?

Reply:
will change 'run' to 'link'